c_api/src/c_api.rs

```diff
@@ -41,6 +41,28 @@ mod cbindgen_fix {
     pub struct C2paReader;
 }
 
+/// The location in which the manifest was read.
+#[repr(C)]
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum C2paManifestLocation {
+    /// The manifest is embedded in the asset.
+    Embedded,
+    /// The manifest is stored remotely.
+    Remote,
+    /// The manifest is stored separately in a sidecar.
+    Sidecar,
```

Comment:
Where is Sidecar used?

Reply:
I should clarify the docs here, `Sidecar` is returned when the manifest is passed explicitly via a method such as `from_manifest_data_and_stream`. If there's a better name to describe this variant let me know.